runtime/calamari/src/lib.rs

```diff
@@ -704,28 +734,6 @@ impl_runtime_apis! {
 	}
 }
 
-// Example custom try-runtime hooks
-pub struct Custom;
-impl OnRuntimeUpgrade for Custom {
```

Comment:
You're going to need these for a runtime upgrade test.

Reply:
Ok, I'll bring it back.